packages/microcosm-www-next/src/pages/index.js

```diff
@@ -21,10 +27,9 @@ export default class IndexPage extends React.Component {
   setVars() {
     this.body = document.body
     this.graphics = document.querySelectorAll('[data-module="ObserveGraphic"]')
-    this.intersectionThreshold = 1.0
     this.observeOptions = {
-      rootMargin: '0px 0px 0px',
-      threshold: this.intersectionThreshold
+      rootMargin: '-76px 0px -145px', //account for height of nav and footer
+      threshold: [0.33, 0.66, 1]
```

Comment:
IntersectionObserver callback will run at different thresholds, which we'll use to fade in/out the graphics appropriately